### DISCUSSION TITLE
Mehrere loadpoints beim mqtt-collector

### DISCUSSION THREAD

**Discussion by deadrabbit87** [solectrus/solectrus#2689] [Q&A]
Hallo, 

ich nutze den mqtt collector in Zusammenhang mit evcc. Soweit funktioniert alles, aber ich habe zwei loadpoints in evcc. Wie sind diese in der `.env` zu konfigurieren?

Danke!

**Comment by ledermann** [comment]
Mehrere Ladepunkte bzw. Wallboxen sind noch nicht implementiert, aber hier ist das schon mal notiert:

- MQTT-Collector: https://github.com/solectrus/mqtt-collector/issues/46
- SENEC-Collector: https://github.com/solectrus/senec-collector/issues/219
- Dashboard: https://github.com/solectrus/solectrus/issues/2241
